### PR TITLE
Add in package/includes to final build product.

### LINKED ISSUE #9: Support extra files?
It looks like the code does not include any "extra" files that aren't typescript.

For example, I have a "templates" directory of handlebars templates that are not included. I see the plugin supports the "exclude" part of the package directive, but nothing for "include".

### PR DESCRIPTION
**Pull request by kopertop**
Resolves #9

### CODE REVIEW

**Comment by kopertop** [comment]
I also locked in fs-p to 2.0.0 because I was getting build errors.

File: src/index.ts [hunk 1]
```diff
@@ -53,6 +55,21 @@ class ServerlessPlugin {
 
     // include node_modules into build
     fs.symlinkSync(path.resolve('node_modules'), path.resolve(path.join(buildFolder, 'node_modules')))
+    // include any "extras" from the "include" section
+    if(this.serverless.service.package.include && this.serverless.service.package.include.length > 0){
+      globby(this.serverless.service.package.include).then((files) => {
```
Comment (schickling) [inline src/index.ts]: Please use `async` & `await` here.

File: src/index.ts [hunk 2]
```diff
@@ -53,6 +55,21 @@ class ServerlessPlugin {
...
+        _.forEach(files, (filename) => {
+          let destFileName = path.resolve(path.join(buildFolder, filename));
```
Comment (schickling) [inline src/index.ts]: Any reason not to use `const` instead of `let`?

File: src/index.ts [hunk 3]
```diff
@@ -53,6 +55,21 @@ class ServerlessPlugin {
...
+          let dirname = path.dirname(destFileName);
+          if(!fs.existsSync(dirname)){
```
Comment (schickling) [inline src/index.ts]: Please use spaces like so: `if (!fs.existsSync(dirname)) {`

(Also applies to other places)

**Comment by schickling** [comment]
I've added a few comments. Would be great if you could adust it :)

**Comment by kopertop** [comment]
No problem, sorry I haven't been writing in TypeScript that long and I only recently learned of Async/Await. Does this look better?

**Comment by schickling** [comment]
Looks great now!

**Pull request merged**